project/src/main/webapp/script.js

```diff
@@ -111,18 +104,32 @@ function getUserLocationFromUi() {
   return coords.lat + "," + coords.lng;
 }
 
+/** Displays a message to the user for a low number of results. */
+function displayNumResultsMessage(numResults) {
+  let messageElement = document.getElementById('problem-message-container');
+  const tryAgainMessage =
+      'You are welcome to try again, and maybe try to change some of the entered parameters.';
+  if (numResults === 0) {
+    messageElement.innerHTML = 'Your search had no results. ' + '<br>' + tryAgainMessage
+  } else if (numResults === 1) {
+    messageElement.innerHTML = 'Your search had only 1 result. ' + '<br>' + tryAgainMessage
+  } else if (numResults === 2) {
+    messageElement.innerHTML = 'Your search had only 2 results. ' + '<br>' + tryAgainMessage
```

Comment:
Include the \<br\> as part of the `tryAgainMessage`?

Reply:
Done